hacking/build_library/build_ansible/filters.py

```diff
@@ -0,0 +1,99 @@
+# Copyright: (c) 2019, Ansible Project
```

Comment:
Probably want a different name here as not all of these are filters.  Probably want to include something in the module name that indicates these are documentation or text-related functions as well.

Reply:
They look like filters, they are all functions passed into the jinja2 template engine to be used with `| <name>` as part of the doc generation.

Reply:
Oh.  heh.  I thought these were just filters as in "sorting and filtering" and saw rst_xline which is a transformation.

So yeah, maybe a jinja2 subdirectory and filters.py in there?

Reply:
Makes sense, has been updated.